mico-admin/src/app/service-detail-overview/service-detail-overview.component.html

```diff
@@ -1,40 +1,23 @@
-<!-- TODO: add buttons, TODO check usage...-->
 <div class="flex">
     <!-- Information Grid-->
     <div class="flex-grow-1 ma2 ml0 flex flex-column">
 
         <div class="relative w-100">
             <ng-container *ngIf="!edit">
                 <!-- display version -->
-                <mat-list dense>
-                    <mat-list-item class="mat-list-item mt2">
-                        <b>VCS Root</b>
-                    </mat-list-item>
-                    <mat-divider></mat-divider>
-                    <mat-list-item>
-                        http://...
-                    </mat-list-item>
-
-                    <mat-list-item class="mat-list-item mt2">
-                        <b>Documentation</b>
-                    </mat-list-item>
-                    <mat-divider></mat-divider>
-                    <mat-list-item>
-                        http://...
-                    </mat-list-item>
+                <!-- TODO change to serviceGET (as soon as api is available -->
+                <mico-data [modelUrl]="'local/servicePUT'" [startData]="service"></mico-data>
```

Comment:
mico-data component needs some css work. Font is really small, and it has too much white space